_posts/2016-08-09-collections-app.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Collecting Connections through the Powerbox"
+author: David Renshaw
+authorUrl: https://github.com/dwrensha
+---
+
+Starting work on a project in Sandstorm often means creating many grains, each pertaining to a
+different aspect of your work. In a typical project, you might have a spreadsheet, a chat room, a
+kanban board, and several source code repositories. Sandstorm makes it easy to share these grains
+with your collaborators, but until recently, you would need to share each grain to each collaborator
+separately --- a task that could quickly become tedious. What if you want to share all of the
+project's grains as a single unit?
+
+Now that we have released the
+[Collections app](https://apps.sandstorm.io/app/s3u2xgmqwznz2n3apf30sm3gw1d85y029enw5pymx734cnk5n78h),
+we have a satisfying answer to that question; to share many grains at once, you add them to a collection.
```

Comment:
That's a pretty neat title. It under-sells one interesting aspect, which is that Collections is a separate app, but maybe that's OK.

You might have space for:

> Share many as one, with the Collections app

![screenshot from 2016-08-09 16-23-58](https://cloud.githubusercontent.com/assets/25457/17533875/b818be94-5e54-11e6-98cc-66084c2b1449.png)

in case you want to draw attention to that technically-interesting aspect (and also user-interesting honestly).